app/models/article.rb

```diff
@@ -0,0 +1,2 @@
+class Article < ApplicationRecord
```

Comment:
add model validations for the title, body and author columns
add presence check , length check for the above columns
add uniqueness check for the author column

Reply:
implemented all the mentioned model validations for title, body and author columns. Please check latest commit